README.md

```diff
@@ -59,6 +59,8 @@ $ scrapy list
 
 Execute spider with name `spider_name`:
 
+> Attention: The command below is slow and will take a while to finish.
```

Comment:
**suggestion**: It is not that the spider is slow, it is because it may take time to scrape the entire website depending on how many items we have available. I would add some ideas how to stop the job running: use [CLOSESPIDER_ITEMCOUNT](https://docs.scrapy.org/en/latest/topics/extensions.html#std-setting-CLOSESPIDER_ITEMCOUNT) or set `start_date` of the spider to a more close date.